src/ng/directive/ngTransclude.js

```diff
@@ -159,35 +159,45 @@
  * </example>
  */
 var ngTranscludeMinErr = minErr('ngTransclude');
-var ngTranscludeDirective = ngDirective({
-  restrict: 'EAC',
-  link: function($scope, $element, $attrs, controller, $transclude) {
+var ngTranscludeDirective = ['$compile', function($compile) {
+  return {
+    restrict: 'EAC',
+    terminal: true,
```

Comment:
I'm mainly concerned about the timing of rendering (so users won't see flashing `{{ ... }}`s).
From what I understand from @dcherman's comment (because I was too lazy to read the source code :stuck_out_tongue:), this hasn't changed. So we're good :+1: